src/storage/mvcc/reader/scanner/mod.rs

```diff
@@ -101,31 +96,25 @@ impl<S: Snapshot> ScannerBuilder<S> {
         let write_cursor_builder = CursorBuilder::new(&self.snapshot, CF_WRITE)
```

Comment:
Maybe move the logic of creating write cursor and lock cursor into the cfg as well, so that they live together?